src/pages/index.js

```diff
@@ -57,9 +147,19 @@ editBtn.addEventListener("click", () => {
 
 // set up popup with form for adding a card
 const cardPopup = new PopupWithForm("#card-modal", ({ title, url }) => {
-  cardSection.addItem({ name: title, link: url });
-  cardPopup.resetForm();
+  renderLoading(true, cardPopup.submitBtn);
+  api
+    .postNewCard({ name: title, link: url })
+    .then((data) => {
+      cardSection.addItem(data);
+      cardPopup.resetForm();
+      renderLoading(false, cardPopup.submitBtn, "Create");
```

Comment:
You need to **only** return the **default** button text in the `finally` block, so that the button could get the text regardless of whether the request succeeds or fails.